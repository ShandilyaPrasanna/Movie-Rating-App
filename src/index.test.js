import React from "react";
import ReactDOM from "react-dom";
import toJson from "enzyme-to-json";
import { shallow, configure } from "enzyme";
import { App } from "./index";
import Adapter from "enzyme-adapter-react-16";

jest.mock("react-dom");

configure({ adapter: new Adapter() });
describe("movieGrid Test", () => {
  let tree;

  beforeEach(() => {
    tree = shallow(<App />);
  });

  it("renders App ", () => {
    expect(toJson(tree)).toMatchSnapshot();
  });
});
