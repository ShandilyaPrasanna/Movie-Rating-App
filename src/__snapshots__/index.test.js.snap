// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`movieGrid Test renders App  1`] = `
<div>
  <div
    className="backgroundImage"
  />
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(MainComponent) />
  </Provider>
</div>
`;
